style.css

```diff
@@ -0,0 +1,102 @@
+@import url('https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,300;1,500&display=swap');
+@import url("color-pallete/color.css");
```

Comment:
Pallete is wrong spelling you any extension spell check right is palette

Reply:
I'm sorry for the mistake.  I'll correct it as soon as possible